zerver/migrations/0419_backfill_message_realm.py

```diff
@@ -14,9 +14,23 @@ def backfill_message_realm(apps: StateApps, schema_editor: BaseDatabaseSchemaEdi
     ArchivedMessage = apps.get_model("zerver", "ArchivedMessage")
     Recipient = apps.get_model("zerver", "Recipient")
     Subscription = apps.get_model("zerver", "Subscription")
+    Stream = apps.get_model("zerver", "Stream")
     UserProfile = apps.get_model("zerver", "UserProfile")
+    Huddle = apps.get_model("zerver", "Huddle")
 
     print()
+
+    print("Deleting dangling Recipient objects.")
```

Comment:
It seems potentially worth extending this to be:
```suggestion
    print("Deleting dangling Recipient objects and their messages, which are inaccessible.")
```

Reply:
Tweaked